packages/blade/src/components/BottomSheet/BottomSheet.web.tsx

```diff
@@ -433,31 +412,43 @@ const _BottomSheet = ({
 
   return (
     <BottomSheetContext.Provider value={contextValue}>
-      <BottomSheetBackdrop zIndex={zIndex} />
-      <BottomSheetSurface
-        {...metaAttribute({ name: MetaConstants.BottomSheet })}
-        {...makeAccessible({ modal: true, role: 'dialog' })}
-        windowHeight={dimensions.height}
-        isDragging={isDragging}
-        style={{
-          opacity: isVisible ? 1 : 0,
-          pointerEvents: isVisible ? 'all' : 'none',
-          height: positionY,
-          bottom: 0,
-          top: 'auto',
-          zIndex,
-        }}
-      >
-        <BaseBox height="100%" display="flex" flexDirection="column">
-          <BottomSheetGrabHandle
-            ref={grabHandleRef}
-            isHeaderFloating={isHeaderFloating}
-            {...metaAttribute({ name: ComponentIds.BottomSheetGrabHandle })}
-            {...bind()}
-          />
-          {children}
-        </BaseBox>
-      </BottomSheetSurface>
+      {isMounted ? (
+        <FloatingFocusManager
+          returnFocus
+          initialFocus={initialFocusRef ?? defaultInitialFocusRef}
+          context={context}
+          modal={true}
```

Comment:
Why do we need to make it `modal=true`? This isn't a modal, right?

Reply:
It is because in the documentation of [FloatingFocusManager](https://floating-ui.com/docs/FloatingFocusManager) the modal prop is there to ensure that the focus remains within the floating component. It does not primarily indicate that the component is modal but the behavior of "focus" is modal.

Reply:
Makes sense